src/workerd/api/tests/compat-flags-test.js

```diff
@@ -0,0 +1,38 @@
+import {
+  ok,
+  strictEqual,
+  throws,
+} from 'node:assert';
+
+import { compatibilityFlags } from 'cloudflare:workers';
+
+const kOn = 'on';
+const kOff = 'off';
+
+export const compatFlagsTest = {
+  test() {
+    throws(() => compatibilityFlags.no_nodejs_compat_v2 = "...");
+    throws(() => compatibilityFlags.not_a_real_compat_flag = "...");
+//    throws(() => { delete compatibilityFlags['nodejs_compat_v2']; });
+
+    strictEqual(compatibilityFlags['nodejs_compat_v2'], kOn);
+    strictEqual(compatibilityFlags['no_nodejs_compat_v2'], kOff);
+    strictEqual(compatibilityFlags['url_standard'], kOn);
```

Comment:
the `wd-test` does not have this compatibility flag, how does this return true?

Reply:
Because it is on by default based on the compat date configured for the worker.